test/suites/integration/bsp/indexer-sanity.test.ts

```diff
@@ -0,0 +1,89 @@
+import assert, { equal, strictEqual } from "node:assert";
```

Comment:
This test suit is better suited in the full net suite

Reply:
Done